components/admin_console/message_export_settings.jsx

```diff
@@ -200,9 +202,9 @@ export default class MessageExportSettings extends AdminSettings {
         }
 
         const dropdownHelpText = (
-            <FormattedHTMLMessage
+            <FormattedMarkdownMessage
                 id='admin.complianceExport.exportFormat.description'
-                defaultMessage='Format of the compliance export. Corresponds to the system that you want to import the data into.<br><br>For Actiance XML, compliance export files are written to the "exports" subdirectory of the configured <a href="/admin_console/files/storage">Local Storage Directory</a>. For Global Relay EML, they are emailed to the configured email address.'
+                defaultMessage='Format of the compliance export. Corresponds to the system that you want to import the data into.\n \nFor Actiance XML, compliance export files are written to the \"exports\" subdirectory of the configured [Local Storage Directory](/admin_console/files/storage). For Global Relay EML, they are emailed to the configured email address.'
```

Comment:
Btw, this link won't work because of relative links in Markdown aren't supported (https://mattermost.atlassian.net/browse/MM-11738), however the old version wasn't much better because it wouldn't work when Mattermost is running in a subpath (like `https://example.com/mattermost`).

Since it's broken either way, would you mind changing this so that it constructs the full URL instead? If you use `getSiteURL()` from `utils/url`, to get the site URL, you can pass it into the `FormattedMarkdownMessage` using values like
```
<FormattedMarkdownMessage
    defaultMessage=` [Local Storage Directory]({siteURL}/admin_console/files/storage)`
    values={{siteURL: getSiteURL()}}
/>
```

Reply:
Thanks @hmhealey! I'll fix it soon.
Other message (`admin.compliance.newComplianceExportBanner`) uses relative path (`../../admin_console/compliance/message_export`) for the same problem. Should I fix it too?

Reply:
Yeah, that one should also be fixed as well. Good catch